packages/react-popover/src/PopoverDescription.md

```diff
@@ -0,0 +1,11 @@
+A `Popover` is used to display content on temporary surface over other content. The position of the `Popover` surface
```

Comment:
I think that it would be better if the md files also have the `.stories.md` post-fix, in order to be clear that these are directly related to storybook. WDYT?

Reply:
just following @PeterDraex example, so I'll give him an opportunity to drop his reasoning.

I think with the similarity to `stories.mdx` extension which actually renders a story it could be confusing. Especially since those files are actually just sugar to store some markdown as a string for parts of docs